frontend/luasettings.lua

```diff
@@ -60,6 +60,10 @@ function LuaSettings:flipTrue(key)
     end
 end
 
+function LuaSettings:replace(table)
```

Comment:
Are you sure? reset() looks like clearing, but this function replaces the content of an LuaSettings into a new table.